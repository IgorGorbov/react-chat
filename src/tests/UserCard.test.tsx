// import * as React from 'react';
// import toJSON from 'enzyme-to-json';
// import { mount } from 'enzyme';
// import UserCard from '../components/ChatCard';
// import UserStatus from '../components/UserStatus';
// import {
//   UserCardContainer,
//   ImageContainer,
//   UserAvatar,
//   UserInfoContainer,
//   UserName,
//   TextMessage,
//   NewCountMessage,
// } from '../styledComponents';
//
// describe('testing ChatsList', () => {
//   const user: IUser = {
//     id: 1,
//     name: 'Louis Litt',
//     avatar: 'https://emilcarlsson.se/assets/louislitt.png',
//     status: Status.Offline,
//   };
//   const wrapper = mount(<UserCard key={user.id} user={user} />);
//
//   it('should contain UserCardContainer', () => {
//     expect(wrapper.find(UserCardContainer)).toHaveLength(1);
//   });
//
//   it('should contain ImageContainer', () => {
//     expect(wrapper.find(ImageContainer)).toHaveLength(1);
//   });
//
//   it('should contain UserAvatar', () => {
//     expect(wrapper.find(UserAvatar)).toHaveLength(1);
//   });
//
//   it('should to hide UserStatus', () => {
//     expect(wrapper.find(UserStatus)).toHaveLength(0);
//   });
//
//   it('should contain UserInfoContainer', () => {
//     expect(wrapper.find(UserInfoContainer)).toHaveLength(1);
//   });
//
//   it('should contain UserName', () => {
//     expect(wrapper.find(UserName)).toHaveLength(1);
//   });
//
//   it('should contain TextMessage', () => {
//     expect(wrapper.find(TextMessage)).toHaveLength(1);
//   });
//
//   it('should contain NewCountMessage', () => {
//     expect(wrapper.find(NewCountMessage)).toHaveLength(1);
//   });
//
//   it('snapshot ChatsList', () => {
//     const tree = toJSON(wrapper);
//     expect(tree).toMatchSnapshot();
//   });
// });
