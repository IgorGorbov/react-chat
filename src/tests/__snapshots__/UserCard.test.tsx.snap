// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing UserList snapshot UserList 1`] = `
<UserCard
  avatar="https://emilcarlsson.se/assets/louislitt.png"
  id={1}
  name="Louis Litt"
>
  <styled.div>
    <div
      className="sc-jzJRlG ikoaaf"
    >
      <styled.div>
        <div
          className="sc-cSHVUG hDUXUW"
        >
          <Styled(Image)
            bordercolor="white"
            bordersize="1px"
            circular={true}
            maxheight="65px"
            maxwidth="65px"
            size="tiny"
            src="https://emilcarlsson.se/assets/louislitt.png"
          >
            <Image
              as="img"
              bordercolor="white"
              bordersize="1px"
              circular={true}
              className="sc-htpNat drfAkD"
              maxheight="65px"
              maxwidth="65px"
              size="tiny"
              src="https://emilcarlsson.se/assets/louislitt.png"
              ui={true}
            >
              <img
                bordercolor="white"
                bordersize="1px"
                className="ui tiny circular image sc-htpNat drfAkD"
                maxheight="65px"
                maxwidth="65px"
                src="https://emilcarlsson.se/assets/louislitt.png"
              />
            </Image>
          </Styled(Image)>
          <styled.span>
            <span
              className="sc-kAzzGY bhplhq"
            />
          </styled.span>
        </div>
      </styled.div>
      <styled.div>
        <div
          className="sc-chPdSV jmNmdm"
        >
          <styled.p>
            <p
              className="sc-kgoBCf eyQkMk"
            >
              Louis Litt
            </p>
          </styled.p>
          <styled.p>
            <p
              className="sc-kGXeez bpwcQb"
            >
              Lorem ipsum dolor sit amet.
            </p>
          </styled.p>
        </div>
      </styled.div>
      <styled.p>
        <p
          className="sc-kpOJdX fuYgIr"
        >
          3
        </p>
      </styled.p>
    </div>
  </styled.div>
</UserCard>
`;
